README.md

```diff
@@ -30,7 +35,17 @@ Your Answer:
 
 ```js
 // your code here
+function Instructor(name){
+
+this.name = name
+givesHomework = function(assignment){
```

Comment:
make sure we attach this function `givesHomework` as a method on `instructor`, like:
```js
this.givesHomework = function(assignment){ .. }
```